### DISCUSSION TITLE
V3.5.0

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#6] [version]
进展列表样式调整，减少层级和颜色。
记本新增「记本图库」功能，可集中展示标准记本、账本、清单本、打卡本的图片，并且可以将这些图片复制到某个文件夹中。
首页负一屏，新增「进展回应」，以回应为主进行展示。
实验室中，取消「日历页-记本标记样式」开关，统一为显示色环；新增「日历色环厚度」，可选薄、中、厚、很厚。
实验室中，取消「进展列表-回应数量」开关，统一改为显示。
首页负一屏，点评集等按钮位置微调。
适配 Android 12，基于 compileSdkVersion 31。实验室新增 Material You Color 查看器。
优化部分页面细节，修复部分反馈过来的闪退问题，提升性能和稳定性。包括文本编辑时不息屏，桌面清单记本插件大小可调，取消资料页的 64 位标识等。

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/'>V3.5.0</a>.</em>

**Comment by laoshuaibip** [comment]
求适配一下横屏
